coming-soon.html

```diff
@@ -0,0 +1,170 @@
+                    </g>
+                </g>
+            </svg>
+            </div>
+
+        </div>
+    </div>
+
+<script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script>
+<script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';fnames[1]='FNAME';ftypes[1]='text';fnames[2]='LNAME';ftypes[2]='text';fnames[3]='BIRTHDAY';ftypes[3]='birthday';}(jQuery));var $mcj = jQuery.noConflict(true);
+</script>
+<script src="node_modules/jquery/dist/jquery.slim.min.js"></script>
+<script src="node_modules/popper.js/dist/umd/popper.min.js"></script>
+<script src="node_modules/bootstrap/dist/js/bootstrap.min.js"></script>
+
+</body>
+</html>
```

Comment:
Technically, the "architecture" of this page is not really good. For this really simple and standalone page, I doubt we need:
- the whole Bootstrap library
- the PopperJs and the jQuery Slim libraries (because there isn't much JavaScript from Bootstrap we use on this page, right)?
- all the "css/style.css" styles for the whole project.

Plus, remember that this is stand-alone page, which should be removed once the project is completed. Therefore, remember or add a TODO to commend out `_coming-soon.scss` when this moment comes.

For now, we can leave it as is. But the next time, I'd expect you to remember our discussion and make it even better ;-)